notebooks/video_clips/VideoClips.ipynb

```diff
@@ -0,0 +1,963 @@
+   "outputs": [],
+   "source": [
+    "# Now we retrieve the items we are working with:\n",
+    "\n",
+    "# Retrieve the YOLO4 interface\n",
+    "\n",
+    "!wget https://raw.githubusercontent.com/drewaogle/YOLOv4-OpenCV-CUDA-DNN/refs/heads/main/yolo4.py\n",
+    "\n",
+    "# Retreive video\n",
+    "!wget https://aperturedata-public.s3.us-west-2.amazonaws.com/aperturedb_applications/norman.mp4\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b5dae152-0643-4701-a505-0b331dd9b204",
+   "metadata": {},
+   "source": [
+    "# Run The Detector\n",
+    "Now that we've downloaded our YOLOv4 code, let's run it.\n",
+    "This will need to download the weights and some configuration; about 300M and will do it automatically.\n",
+    "\n",
+    "After downloading or verify files, it will then process the video. with `no_squash_detections` as `True` it won't overwrite an existing output dir, so delete it to rerun. This code can support hardware acceleration, but is designed so it won't be unwieldly without it. Detections should be at about 3-10fps without hardware, and take less 5 minutes.\n",
+    "\n",
+    "If a file were to fail halfway through it, rerunning the loader wont be happy ( sha2 sums won't match )\n",
+    "`!rm ~/models` will reset the downloads, though."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1fda14d1-dc17-48da-af51-bdeb8d6e468d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from yolo4 import RemoteYOLOv4\n",
+    "# options for detector\n",
+    "class DetectorOptions:\n",
+    "    image='' # path for images\n",
+    "    stream='' # path for stream\n",
+    "    cfg=\"models/yolov4.cfg\" # path to config\n",
+    "    weights=\"models/yolov4.weights\" # path to weights\n",
+    "    namesfile=\"models/coco.names\" # path for output to name mapping\n",
+    "    input_size=416\n",
+    "    use_gpu=False # use GPU or not\n",
+    "    outdir=\"output/norman\"\n",
+    "    no_squash_detections=True # if detections exist, don't rerun.\n",
+    "    def __init__(self, image='',stream=''):\n",
+    "        self.image = image\n",
+    "        self.stream=stream # 'webcam' to open webcam w/ OpenCV\n",
+    "\n",
+    "# now we pull data, and and run detection.\n",
+    "dopts = DetectorOptions( stream=\"norman.mp4\")\n",
+    "yolo = RemoteYOLOv4.__new__(RemoteYOLOv4)\n",
+    "yolo.__init__(dopts)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fa362d75-8c06-4adb-a0ba-3b66421a1807",
+   "metadata": {},
+   "source": [
+    "## Now Check Detections\n",
+    "the YoloV4 code we use outputs detections sequentially into a csv file, so let's load the file and see what the output looks like."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e3053bdb-a78a-4a8e-a454-2507a1477eb9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "df = pd.read_csv(\"output/norman/detections.csv\")\n",
```

Comment:
Could you run this code or show a snippet? It's nice to see the output right away